docs/misc/dvd-remux.md

```diff
@@ -0,0 +1,670 @@
+            but it's been known
+            to create broken remuxes from time to time.
+            Use this method
+            at your own risk!
+
+        !!! warning "Input file"
+            MakeMKV requires a DVDISO
+            or physical disc
+            to work.
+            If you only have an unpacked DVDISO,
+            you will need to use a different method.
+
+        MakeMKV can be downloaded from the following link:
+
+        - [MakeMKV](https://www.makemkv.com/)
+
+        Once you have downloaded and installed MakeMKV,
+        you will need to register it.
+        A free key can be found [here](https://forum.makemkv.com/forum/viewtopic.php?t=1053).
+        Once you have registered it,
+        you will need to restart the program.
+
+        !!! example "Selecting titles"
+
+            === "ISO"
+
+                Click the "Open File" button
+                and select your DVDISO.
+                It will start processing the file,
+                and then show you this window:
+
+                ![MakeMKV main window](./img/dvd-remux/makemkv/main-window.png)
+
+                Select the titles you want to remux,
+                give it an output folder and name,
+                and click the button underneath "Make MKV".
+
+                ![MakeMKV Progress window](./img/dvd-remux/makemkv/progress-window.png)
+
+                Once it's done,
+                the files will be in the output folder.
+
+                ![MakeMKV output folder](./img/dvd-remux/makemkv/output.png)
+
+            === "Physical disc"
+
+                Insert your physical disc.
+                MakeMKV will automatically detect it,
+                and then show you this window:
+
+                ![MakeMKV main window](./img/dvd-remux/makemkv/inserted-disc-window.png)
+
+                Click the "Backup" button,
+                select "Decrypt video files"
+                (if possible),
+                pick an output directory,
+                and press "OK".
+                It will now start creating
+                a back-up of your disc.
+
+                ![MakeMKV backup window](./img/dvd-remux/makemkv/copy-dvd-window.png)
+
+                Once it's done,
+                [follow the instructions
+                for the ISO method](#__tabbed_5_1).
+
+## Setting the correct aspect ratio
+
+DVDs (and later SD Blu-rays)
+are stored in a format known as "anamorphic video".
+This means that the stored video dimensions
+differ from the intended display dimensions.
+A typical NTSC DVD stores video at 720x480 pixels
+(a 3:2 aspect ratio),
+but displays it at either widescreen (16:9)
+or standard definition (4:3) aspect ratios.
+
+To achieve the correct display size,
+DVDs make use of a [Sample Aspect Ratio (SAR)](https://en.wikipedia.org/wiki/Sample_aspect_ratio),
+also known as a [Pixel Aspect Ratio (PAR)](https://en.wikipedia.org/wiki/Pixel_aspect_ratio).
+This value tells the player how much to stretch each pixel.
```

Comment:
Maybe write out the formulas here for how they relate to one another.

Reply:
I have trouble writing out this part. Can I leave it to you?